docs/csharp/language-reference/keywords/interpolated-strings.md

```diff
@@ -14,7 +14,7 @@ ms.author: "wiwagn"
 ---
 # Interpolated Strings (C# Reference)
 
-Used to construct strings.  An interpolated string looks like a template string that contains *interpolated expressions*.  An interpolated string returns a string that replaces the interpolated expressions that it contains with their string representations.  
+Used to construct strings.  An interpolated string looks like a template string that contains *interpolated expressions*.  An interpolated string returns a string that replaces the interpolated expressions that it contains with their string representations. This features is available in C# 6 and newer.
```

Comment:
features --> feature
newer --> later

Reply:
I'd probably change newer to "later versions" instead.